benchmarks/benchmarks/bench_import.py

```diff
@@ -13,7 +13,7 @@ class Import(Benchmark):
     def execute(self, command):
         call((executable, '-c', command))
 
-    def time_numpy(self):
+    def time_import_numpy(self):
```

Comment:
I think this breaks continuity of the benchmarks and is unnecessary, can you undo this change?

Reply:
yes, sorry, i thought this was going to my own branch....

Reply:
For now, I'll just leave it because I can select just the benchmark I want to run locally:
```
asv run -b time_import_numpy  master..improve_import_time -E conda:3.7
```

unless there is an other way for me to select `time_numpy` without the other tests in there.